react/utils/assets.js

```diff
@@ -0,0 +1,100 @@
+function isRelative(path) {
+  return path[0] !== '/'
+}
+
+function prefix(path) {
+  const prefix = isRelative(path) ? '/assets/' : ''
```

Comment:
These `prefix` and `isRelative` are legacy code that does not make sense anymore. It also breaks the dynamic locales loading, take a look here https://github.com/vtex-apps/render-runtime/pull/11

Reply:
Thanks! Will remove.